packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -311,12 +372,17 @@ class BundleToolArtifact extends CachedArtifact {
   }
 
   @override
-  String? get checksum =>
+  String? get sha256Checksum =>
       // SHA-256 checksum of the bundletool.jar file.
       // When updating the bundletool version, be sure to update this checksum.
       // This can be done by running the following command:
       // ```shell
       // shasum --algorithm 256 /path/to/file
       // ```
+      // TODO(bryanoltman): github includes `content-md5` in the response
```

Comment:
That is insteresting is that new? Or something undocumented? I remember when I implemented this manually generated hash I had found an issue from GitHub that its release artifact didn't had any hash 🧐

Anyway totally agree that we should prefer the value on the hash is there is one!!!

Reply:
Not sure if it's new or undocumented or what, but if you follow the redirect (which points to https://objects.githubusercontent.com/github-production-release-asset-2e65be/132104296/9b72e02c-edf3-4618-a84f-869ac3423e42?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20240712%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240712T001036Z&X-Amz-Expires=300&X-Amz-Signature=aa5aadcee7d2a327e9643b1d17350416575f1ead1110275ed5452bd15187d188&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=132104296&response-content-disposition=attachment%3B%20filename%3Dbundletool-all-1.15.6.jar&response-content-type=application%2Foctet-stream) and make a HEAD request to that, you'll see in the headers:

```
content-md5: 2RIwqYbUqZDnu2k1G56UZA==
```

Reply:
Cool! I found the issue I mentioned: https://github.com/orgs/community/discussions/23512

Anyhow, was just curious, glad there is a better solution than manually generating

Reply:
Looks like content-md5 is even a standardized header! (note the lack of x- prefix): https://stackoverflow.com/questions/8300471/is-content-md5-field-in-the-http-response-universal

Reply:
Basically we're just sitting on top of the existing network infrastructure (which is already delivering us a correct binary) to then later verify again that the binary is still OK during usage, I think?